potodo/_po_file.py

```diff
@@ -0,0 +1,73 @@
+import polib
+
+from pathlib import Path
+
+
+class PoFile:
+    """
+    Class for each `.po` file containing all the necessary information about its progress
+    """
+
+    def __init__(self, path: Path, directory: str):
+        """
+        Initializes the class with all the correct information
+        """
+        self.path: Path = path
+        self.filename: str = path.name
+        self.pofile: polib.POFile = polib.pofile(self.path)
+        self.directory: str = directory
```

Comment:
Can't you get directory from the path? I think it's `.parent`.

Reply:
`parent.name` yes 👍